wandb/sdk/wandb_settings.py

```diff
@@ -241,7 +263,17 @@ def _validate(self, value: Any) -> Any:
             )
             for v in _validator:
                 if not v(value):
-                    raise ValueError(f"Invalid value for property {self.name}: {value}")
+                    # fixme: this is a temporary measure to bypass validation of certain settings
+                    #  remove this once we are confident
+                    if self.name in self.__strict_validate_settings:
+                        raise ValueError(
+                            f"Invalid value for property {self.name}: {value}"
+                        )
+                    else:
+                        wandb.termwarn(
+                            f"Invalid value for property {self.name}: {value}. "
+                            "This will cause an error in the future."
+                        )
```

Comment:
should you continue running more validators or should you break/return?

Reply:
right, breaking now.